docusaurus.config.ts

```diff
@@ -39,8 +39,8 @@ const docsConfigBase = {
 } satisfies DocsPlugin.Options;
 
 const config: Config = {
-  title: 'THEOplayer Documentation',
-  tagline: 'Your one-stop shop for everything THEOplayer.',
+  title: 'THEOdocs',
```

Comment:
Nitpick: The logo uses THEO Docs (capital D). Do we want to align them?

Reply:
Marketing said they were going to fix the logo to have a lowercase d. 😉